### PR TITLE
SJQ2T-314: BE - Access setup

### PR DESCRIPTION
**Pull request by ivanfhtec**
* flyway, entities, and repositories for access/permission management

* shorter names for action enums

### CODE REVIEW

**Comment by ivanfhtec** [comment]
image

**Comment by ivanfhtec** [comment]
types of access:
* view

* download

* edit

* owner

File: backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.htecgroup.skynest.model.entity;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public enum AccessType {
```
Comment (petarstoimenovskihtec) [inline backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java:8]: Where do you use this enum?

File: backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  VIEW("view"),
+  DOWNLOAD("download"),
+  EDIT("edit"),
+  OWNER("owner");
+  public final String text;
```
Comment (petarstoimenovskihtec) [inline backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java]: probably this needs to be closed for accessing , or provide a getter if needed

**Review by petarstoimenovskihtec — APPROVED** [review]
With just few small comments

File: backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java (hunk 1 above)
Comment (ivanfhtec) [reply backend/src/main/java/com/htecgroup/skynest/model/entity/AccessType.java:8]: nowhere at the moment, but I'm sure whoever writes the permissions service will need this enum

**Pull request merged**